lifetimes/utils.py

```diff
@@ -197,9 +199,12 @@ def summary_data_from_transaction_data(transactions, customer_id_col, datetime_c
     datetime_format: string, optional
         a string that represents the timestamp format. Useful if Pandas can't understand
         the provided format.
-    freq: string
+    freq: string, optional
         Default 'D' for days, 'W' for weeks, 'M' for months... etc. Full list here:
         http://pandas.pydata.org/pandas-docs/stable/timeseries.html#dateoffset-objects
+    div_coef: float, optional
```

Comment:
Can you think of a better name for `div_coef`? Something more representative?

Reply:
What about `freq_multiplier`? Or maybe you could suggest a better one?